### ISSUE TITLE
Image Width

### ISSUE THREAD

**Issue by BurakParsAydin** [kenwheeler/slick#719]
![untitled-1](https://cloud.githubusercontent.com/assets/6393475/4800592/d07f4fe0-5e27-11e4-856d-9573e4ff695b.png)

![untitled-2](https://cloud.githubusercontent.com/assets/6393475/4800629/3811c656-5e28-11e4-8f81-474423d2affa.png)

Hello,

I want to make my slider like first image, but i hadn't. It appears like second image. So images overflow each other. By the way, i use `transform:scale(2)` to enlarge the images. 

In summary, how to build slider like first image ? 

Thanks in advance
